pydantic/validators.py

```diff
@@ -218,6 +219,22 @@ def decimal_validator(v: Any) -> Decimal:
     return v
 
 
+def ip_v4_address_validator(v: Any) -> IPv4Address:
+    if isinstance(v, IPv4Address):
+        return v
```

Comment:
no test for this case

Reply:
Oh, you're right! I've fixed both cases. Now coverage's back to almost 100%